src/Whoops/Util/TemplateHelper.php

```diff
@@ -72,6 +72,39 @@ public function escapeButPreserveUris($raw)
         );
     }
 
+    /**
+     * Makes sure that the given string breaks on the delimiter.
+     *
+     * @param  string $delimiter
+     * @param  string $s
+     * @return string
+     */
+    public function breakOnDelimiter($delimiter, $s)
+    {
+        $parts = explode($delimiter, $s);
+        foreach ($parts as &$part) {
```

Comment:
I thought we don't use by-reference for a while now. Does this have some interesting meaning I am missing?